app/assets/stylesheets/application.scss

```diff
@@ -74,11 +75,11 @@ $search-result: #008cba;
 
   .submit-form {
     background: rgba(0, 0, 0, 0.6);
-    -moz-border-radius: 15px;
-    -webkit-border-radius: 15px;
-    border-radius: 15px;
     color: $white;
-    margin-top: 10px;
+    font: bold 12px/24px Helvetica, Sans-Serif;
+    padding: 5px;
+    padding-right: 10px;
```

Comment:
Properties should be ordered background, color, font, padding, padding-left, padding-right